### ISSUE TITLE
Does it support for multi-label classification

### ISSUE THREAD

**Issue by xdwang0726** [IBM/WordMoversEmbeddings#2]
Hi, I just wondering does it support for multi-label classification as in the example, the pre-processed dataset seems to fit for a binary classification task. I am wondering, if I want to use the code for multi-label classification, how could I deal with the "doc1_label_ID" in the prepared text file. Thank you in advance for your help!

**Comment by teddylfwu** [comment]
Yes, it is able to support multi-label classification with very minor modification in the training classifier. The WME is essentially providing only the embeddings of each sentence/document and can be combined with any classifier/clustering task.
